packages/synapse-react-client/src/lib/containers/oauth/OAuthManagement.tsx

```diff
@@ -46,13 +48,15 @@ export const OAuthManagement: React.FunctionComponent = () => {
   }
 
   return (
-    <div className="bootstrap-4-backport OAuthEditor">
+    <div className="bootstrap-4-backport">
```

Comment:
Table styling requires `bootstrap-4-backport` class -- convert buttons within table to MUI, but defer updating Table to MUI components to a future PR

main | feature
:---:|:---:
<img width="999" alt="main_OAuthManagement" src="https://user-images.githubusercontent.com/26949006/234957892-25380b4d-86e9-4ab4-8515-893ad8a5acdc.png"> | <img width="997" alt="feature_OAuthManagement" src="https://user-images.githubusercontent.com/26949006/234957873-f1ceb25e-f098-4101-8927-55aba1234050.png">

Reply:
Yeah, that's fine with me. Tables will be a little trickier. Might need dedicated design input plus it could be worth looking for a well-maintained, one-size-fits-all soln for the more complex use cases like in SynapseTable, DatasetItemsEditor, EntityFinder